brainglobe_utils/image_io/load.py

```diff
@@ -86,9 +86,9 @@ def load_any(
     np.ndarray
         The loaded brain.
     """
-    src_path = str(src_path)
+    src_path = Path(src_path)
```

Comment:
It's times like this when I wish Python's type-hinting actually enforced casting when passing inputs :joy:

Reply:
Me too! :sweat_smile: